otx/algorithms/detection/adapters/mmdet/models/dense_heads/mmov_yolov3_head.py

```diff
@@ -10,14 +11,19 @@
 from mmdet.models.builder import HEADS
 from mmdet.models.dense_heads.yolo_head import YOLOV3Head
 
-from ...mmov_model import MMOVModel
+from otx.mpa.modules.ov.models.mmov_model import MMOVModel
+
+# TODO: Need to fix pylint issues
+# pylint: disable=too-many-instance-attributes, keyword-arg-before-vararg
 
 
 @HEADS.register_module()
 class MMOVYOLOV3Head(YOLOV3Head):
+    """MMOVYOLOV3Head class for OTX."""
+
     def __init__(
         self,
-        model_path_or_model: Union[str, ov.Model],
+        model_path_or_model: Union[str, ov.Model] = None,
```

Comment:
I think we should add Optional here. If I'm wrong just resolve this

Reply:
Um, yeah, let me do it a different way.

Reply:
This argument was not optional. Why `None`?

Reply:
Looks like my mistake, I'll revert.

Reply:
Fixed.